tools/SetupFlow/DevHome.SetupFlow/Services/SetupFlowOrchestrator.cs

```diff
@@ -171,12 +170,53 @@ private bool CanGoToPreviousPage()
         return HasPreviousPage && CurrentPageViewModel.CanGoToPreviousPage;
     }
 
+    // Teaching tip needs to be open and visible to show up.
+    [ObservableProperty]
+    private bool _isTeachingTipOpen;
+
+    [ObservableProperty]
+    private bool _isTeachingTipVisible;
+
     [RelayCommand(CanExecute = nameof(CanGoToNextPage))]
     public async Task GoToNextPage()
     {
+        IsTeachingTipVisible = false;
+        IsTeachingTipOpen = false;
         await SetCurrentPageIndex(_currentPageIndex + 1);
     }
 
+    /// <summary>
+    /// Mimics the tooltip "show" behavior by waiting on a pointer hover.
+    /// </summary>
+    /// <returns>A task.</returns>
+    [RelayCommand]
+    public async Task StartTeachingTipShowTimer()
+    {
+        await Task.Delay(1500);

```

Comment:
I could be wrong but I think the methods being asynchronous may be the reason why Im seeing this behavior?

https://github.com/microsoft/devhome/assets/105318831/0265a597-9ff2-4806-937d-60939d95ea13

Everything looks good until we add a repo and quickly switch between the next and previous buttons. At some point we end up with an empty tooltip. Not a major issue since I had to really switch back and forth fast for it to happen.

however, the last one where I cancel the flow and go back to the main page looks like the tool tip still appears. 

Also when clicking the x, and then moving back to the previous page is the tool tip suppose to pop back up? Thats the behaviour Im seeing, just wanted to confirm if thats what you intended to happen.